src/main/java/net/twasi/obsremotejava/objects/Scene.java

```diff
@@ -13,4 +14,16 @@ public String getName() {
     public List<Source> getSources() {
         return sources;
     }
+
+    public List<Source> getAllSources() {
```

Comment:
Could you add javadocs here explaining the difference between these two methods? 
Or rename it to something like "getSourcesIncludingGroupChildren"?